### ISSUE TITLE
Keep a list of downloads on Gitub that the tool uses to get where things are

### ISSUE THREAD

**Issue by donker** [nvisionative/nvQuickSite#128]
Instead of a fixed list keep a Json file on Github that the tool consults to get a list of the following:
1. Object name (e.g. DNN Platform),
2. Version,
3. Download url,
This file will be maintained by us but means users won't need to update the tool.

**Comment by nvisionative** [comment]
@donker let us know if we need to discuss this further prior to implementation.  We'd like to get another release out soon with this rolled in.  Thanks!

**Comment by iJungleboy** [comment]
Just a note: if we use github, I believe you should use https://rawgit.com/ as the url as github doesn't like being a CDN - but I may also be wrong.

**Comment by nvisionative** [comment]
Great suggestion @iJungleboy - that looks like a good solution, but is it a **reliable** free service?  If not, perhaps we should reference the GitHub file link directly.

**Comment by iJungleboy** [comment]
Since nvQuickSite is a windows application, I guess you can go either way. For web applications it's often the only way to do it, because github gives wrong mime-types for files, so that browsers can't use them as CDNs. nvQuickSite can always ignore the mime type, in which case I guess it's ok.

**Comment by nvisionative** [comment]
Just a quick update on this...we have completed testing of the code and everything looks good with the new model @donker and I came up with.  Also @iJungleboy, it appears we will not need to utilize https://rawgit.com/.  We'll double check during final testing, prior to release, but initial tests look good accessing a GitHub file URL from within the context of the windows app.  Now for the hard part - updating the packages.json file to point to all the correct versions of DNN.  Until DNN Corp is able to get the DNN.Platform repo releases up-to-date, we'll just point to our current server repo of the install packages.  Thanks everyone!
